plugins/network-elements/nuage-vsp/src/com/cloud/agent/api/guru/TrashNetworkVspAnswer.java

```diff
@@ -31,4 +31,14 @@ public TrashNetworkVspAnswer(Command command, Exception e) {
     public TrashNetworkVspAnswer(TrashNetworkVspCommand cmd, boolean success, String details) {
         super(cmd, success, details);
     }
+
+    @Override
+    public boolean equals(Object obj) {
+        return super.equals(obj);
+    }
+
+    @Override
+    public int hashCode() {
+        return super.hashCode();
+    }
```

Comment:
Why does this class exist?  It adds no state or behavior to the ``Answer`` base class.  Why not simply use ``Answer`` in this circumstance?